ocaml/license/jbuild

```diff
@@ -2,17 +2,13 @@
  ((name daily_license_check_main)
   (public_name daily-license-check)
   (package xapi)
+  (modules (:standard \ test_daily_license_check))
```

Comment:
Would building that test require adding back these dependencies, or does the test simply not work?

Reply:
From what I could see the test has not been built or run for a while, I did not check what happens running it